### DISCUSSION TITLE
Are fragments used anyway?

### DISCUSSION THREAD

**Discussion by icamps** [grimme-lab/xtb#712] [Q&A] [answered]
Hello,

Some of my system are formed by more than one molecule. In this case, `xTB` correctly identify the current fragments.

My question: Can `xTB` do any treatment/calculation using this information about having a structure with 2 fragments?

Thanks.

**Comment by cplett** [comment, marked as answer]
Hi,
Currently, the fragmentation that is printed at the beginning is only used for constraints.

**Reply by John-zzh** [reply]
Hi there,

I'm also dealing with a system with >=2 fragments,  can I define the fragmentation? I think gfnff sometime mis-judge the fragmentation. 
for example,  I want atom 1-20 for fragment 1,  atom 21-100 for fragment 2.
